assets/scss/_variables_project.scss

```diff
@@ -117,34 +118,56 @@ body {
         background-color: $dark-bg-primary;
     }
 }
-p, td, div
+
+p, td, div, li, .lead
 {
-    font-family: "Source Sans Pro", sans-serif;
+    font-family: $base-font;
+    font-weight: $font-weight-regular !important;
+    font-size: 16px;
+    [data-bs-theme=dark] & {
+        color: $dark-text-primary;
+    }
 }
+
 .td-sidebar-nav__section-title, .td-sidebar-nav__section without-child 
 {
-    font-size: 16px; 
+    font-size: 14px; 
     font-weight: $font-weight-bold;
 }
 
 // Headings
 h1, h2, h3, h4, h5, h6 {
-    font-weight: $font-weight-regular !important;
-    font-family: "Source Serif 4", sans-serif;
     color: $text-primary;
-
     [data-bs-theme=dark] & {
-        color: $dark-text-primary;
+        color: white !important;
     }
 }
 
 // Heading sizes
-h1, .td-content h1 { font-size: 32px !important; }
-h2, .td-content h2 { font-size: 24px !important; }
-h3, .td-content h3 { font-size: 20px !important; }
-h4, .td-content h4 { font-size: 18px !important; }
-h5, .td-content h5 { font-size: 17px !important; }
-h6, .td-content h6 { font-size: 16px !important; }
+h1, .td-content h1 { 
+    font-size: 32px !important;
+    font-weight: $font-weight-bold !important;
+}
+h2, .td-content h2 { 
+    font-size: 24px !important;
+    font-weight: $font-weight-bold !important;
+}
+h3, .td-content h3 { 
+    font-size: 20px !important;
+    font-weight: $font-weight-bold !important;
+}
+h4, .td-content h4 { 
+    font-size: 16px !important;
+    font-weight: $font-weight-bold !important;
+}
+h5, .td-content h5 {
```

Comment:
This is now the same as the base font, which means that H6 is smaller than the base font. WDYT?